account_reconcile_oca/views/account_move_line.xml

```diff
@@ -40,7 +40,7 @@
                     <field
                     name="name"
                     string="Journal Item"
-                    filter_domain="[                         '|', '|', '|',                         ('name', 'ilike', self), ('ref', 'ilike', self), ('account_id', 'ilike', self), ('partner_id', 'ilike', self)]"
+                    filter_domain="['|','|','|','|','|','|',('name', 'ilike', self), ('ref', 'ilike', self), ('account_id', 'ilike', self), ('partner_id', 'ilike', self),('amount_residual', 'ilike', self),('amount_residual_currency', 'ilike', self),('discount_amount_currency', 'ilike', self)]"
```

Comment:
`discount_amount_currency` seems not very relevant

Reply:
Done!